Merge ownership update on release post script (#1004)
sourcegraph/handbook: content/marketing/product-marketing/release_post_process.md

You can tell when the release is cut by following along in #progress.

> - The marketing release post owner runs the script to generate the changelog (being sure to replace the version number in the command), and merges it into the release template at the end of the release post ([loom guide](https://www.loom.com/share/59da6bc1784a48e9b6af4d9e620ee4df)): `go run ./bin/generate_changelog_items.go -versions 3.28 -i ../sourcegraph/CHANGELOG.md`.

Author: ```suggestion
- The marketing release post owner runs the command to generate the changelog (being sure to replace the version number in the command), and then pastes the output at the end of the release post ([loom guide](https://www.loom.com/share/59da6bc1784a48e9b6af4d9e620ee4df)): `go run ./bin/generate_changelog_items.go -versions 3.28 -i ../sourcegraph/CHANGELOG.md`.
```
